Week1/homework/index.html

```diff
@@ -1 +1,23 @@
-<!-- replace this with your HTML content -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li>About</li>
```

Comment:
Do we need this navigation list for About and Contact here?

Reply:
@yash-kapila Thank you for having time to review my homework, much appreciated!
I'm not sure if I got you well! Do you mean using About and Contact inside li is not correct or we don't need to use it here at all? Could you please clarify what you mean?

Reply:
I don't think we need them at all for this homework.